components/script/dom/htmlformelement.rs

```diff
@@ -465,6 +466,22 @@ impl HTMLFormElement {
     /// https://html.spec.whatwg.org/multipage/#statically-validate-the-constraints
     fn static_validation(&self) -> Result<(), Vec<FormSubmittableElement>> {
         let node = self.upcast::<Node>();
+        for child in node.traverse_preorder() {
+            if let Some(el) = child.downcast::<Element>() {
+                if el.disabled_state() {
+                    continue;
+                }
+                let validatable = match el.as_maybe_validatable() {
+                    Some(v) => v,
+                    None => continue,
+                };
+                if !validatable.is_instance_validatable() {
+                    continue;
+                }
+                // pass empty flags initially and set appropriate flags according to the type of element called
+                validatable.validate(ValidationFlags::empty());
+            }
+        }
```

Comment:
Please push the code to a git branch somewhere, and link to it (as well as mentioning the error text) here. It's also fine if you just push it as a new commit on this same pull request.